packages/nimble-components/src/table-column/number-text/models/no-unit-scale-formatter.ts

```diff
@@ -0,0 +1,20 @@
+import { IntlNumberFormatScaledUnit } from './intl-number-format-scaled-unit';
+import type { ScaledUnit } from './scaled-unit';
+import { UnitScaleFormatter } from './unit-scale-formatter';
+
+/**
+ * Degenerate UnitScaleFormatter for formatting without units
+ */
+export class NoUnitScaleFormatter extends UnitScaleFormatter {
```

Comment:
No is a weird description. What kind of unit is this, a "No" unit. I'd vote for AbsentUnitScaleFormatter or EmptyUnitScaleFormatter or UnitlessUnitScaleFormatter